lib/src/components/NxCombobox/NxCombobox.scss

```diff
@@ -45,27 +43,9 @@
     text-align: center;
   }
 
-  &--dropdown-showable {
-    @mixin show-dropdown {
-      .nx-combobox__menu {
-        display: block;
-        position: absolute;
-        right: initial;
-  
-        .nx-dropdown-button.selected {
-          background-color: var(--nx-swatch-blue-90);
-        }
-      }
-  
-      .nx-combobox__alert {
-        display: block;
-        position: absolute;
-        right: initial;
-      }
-    }
-
-    &:focus-within {
-      @include show-dropdown;
+  &:focus-within {
+    .nx-dropdown-button.selected {
+      background-color: var(--nx-swatch-blue-90);
```

Comment:
I feel like this probably doesn't need to be inside of `:focus-within`

Reply:
> I feel like this probably doesn't need to be inside of `:focus-within`

removed here: [3246834](https://github.com/sonatype/sonatype-react-shared-components/pull/876/commits/3246834ecfce508514dfa70948b9035699f52413)